portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,59 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Jacob's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="about-me">
+        <h1>Hi, I am Jacob. Welcome to my personal portfolio page.</h1>
```

Comment:
Consider keeping lines to 80 or 100 chars (you can use an auto formatter) so that it's easier to view on Github :)